legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-core/src/main/java/org/finos/legend/engine/persistence/components/relational/api/RelationalSinkCleanerAbstract.java

```diff
@@ -161,57 +157,19 @@ private Operation buildDropPlanForLockTable()
         return Drop.of(true, lockInfoDataset.get(), false);
     }
 
-    private LogicalPlan buildLogicalPlanForMetadataCleanup()
-    {
-        List<Operation> operations = new ArrayList<>();
-        operations.add(buildDeleteCondition());
-        return LogicalPlan.of(operations);
-    }
-
-    private Operation buildDeleteCondition()
-    {
-        StringValue mainTableName = getMainTable();
-        FieldValue tableNameFieldValue = FieldValue.builder().datasetRef(metadataDataset().get().datasetReference()).fieldName(metadataDataset().tableNameField()).build();
-        FunctionImpl tableNameInUpperCase = FunctionImpl.builder().functionName(FunctionName.UPPER).addValue(tableNameFieldValue).build();
-        StringValue mainTableNameInUpperCase = StringValue.builder()
-                .value(mainTableName.value().map(field -> field.toUpperCase()))
-                .alias(mainTableName.alias()).build();
-        Condition whereCondition = Equals.of(tableNameInUpperCase, mainTableNameInUpperCase);
-        return Delete.of(metadataDataset().get(), whereCondition);
-    }
-
-    private StringValue getMainTable()
-    {
-        return StringValue.of(mainDataset().datasetReference().name().orElseThrow(IllegalStateException::new));
-    }
-
     private SinkCleanupIngestorResult executeSinkCleanup(SinkCleanupGeneratorResult result)
     {
         SinkCleanupIngestorResult ingestorResult;
         try
         {
             executor.executePhysicalPlan(result.dropSqlPlan());
-                try
-                {
-                    executor.begin();
```

Comment:
now that we are not calling executor.begin(), we dont need the finally block anymore i believe